changelog.d/8784.misc

```diff
@@ -0,0 +1 @@
+Add additional type hints in the registration and SSO handlers.
```

Comment:
Should this be a bugfix now?

Reply:
Good call! I'll re-word this to talk about the bug being fixed.

Reply:
I updated this although I don't love my wording...